### DISCUSSION TITLE
make kind quickstart  failed~~~

### DISCUSSION THREAD

**Discussion by jingchu000** [kubewharf/kelemetry#119] [Q&A] [answered]
Have a nice day! 👋
docker-compose -f quickstart.docker-compose.yaml \
                -f <(jq -n --arg KELEMETRY_IMAGE "" "$QUICKSTART_JQ_PATCH") \
                up --no-recreate --no-start
kubectl config view --raw --minify --flatten --merge >hack/client-kubeconfig.local.yaml
sed -i "s/0\.0\.0\.0/$(docker network inspect kelemetry_default -f '{{(index .IPAM.Config 0).Gateway}}')/g" hack/client-kubeconfig.local.yaml
sed: 1: "hack/client-kubeconfig. ...": extra characters at the end of h command
make: *** [quickstart] Error 1

我的机器：MacBook Pro Apple M2

**Comment by SOF3** [comment, marked as answer]
The `make kind` script was only tested on Linux with GNU coreutils. It appears that MacOS has slightly different syntax <https://stackoverflow.com/a/44864004/3990767>. You may try to edit this line to `sed -i.bak` as per the stackoverflow answer.

**Comment by SOF3** [comment]
By the way, currently we do not distribute arm64 docker images for Kelemetry yet. You may have to build the image yourself first:

```
docker build -t kelemetry .
make quickstart KELEMETRY_IMAGE=kelemetry
```
